Mathlib/Topology/Basic.lean

```diff
@@ -519,6 +519,13 @@ theorem closure_iUnion_of_finite [Finite ι] (f : ι → Set X) :
   rw [← sUnion_range, (finite_range _).closure_sUnion, biUnion_range]
 #align closure_Union closure_iUnion_of_finite
 
+@[simp]
+theorem closure_iUnion₂_of_finite {n : ℕ} (f : ℕ → Set X) :
```

Comment:
This is not a good name, it is not specific enough. Maybe `closure_iUnion₂_le_nat`?

Reply:
870a04d